src/api/tests/participantsRoute.test.ts

```diff
@@ -0,0 +1,84 @@
+import { QueryBuilder } from 'objection';
+import request, { Request } from 'supertest';
+
+import api from '../api';
+import { Participant } from '../entities/Participant';
+import { User } from '../entities/User';
+import useTestServer from './utils';
+
+const mockParticipant = (participant?: Participant | null) => {
+  jest.spyOn(Participant, 'query').mockReturnValueOnce(
+    QueryBuilder.forClass(Participant).resolve(
+      participant === undefined
+        ? {
+            id: '1',
+            name: 'Test Participant',
+            location: 'Test Location',
+            allowSharing: true,
+          }
+        : participant
+    )
+  );
+};
+
+const mockUser = (user?: User | null) => {
+  jest.spyOn(User, 'query').mockReturnValueOnce(
+    QueryBuilder.forClass(User).resolve(
+      user === undefined
+        ? {
+            id: '1',
+            email: 'test_user@example.com',
+            name: 'Test User',
+            participantId: 1,
+          }
+        : user
+    )
+  );
+};
+describe('Participants Route tests', () => {
+  const withToken = useTestServer();
+
+  describe('hasParticipantAccess middleware', () => {
+    afterEach(() => {
+      jest.restoreAllMocks();
+    });
+
+    it('should allow access to an authenticated user with permission', async () => {
```

Comment:
This test doesn't seem to be named properly, and I'm not sure what it's achieving. It's looking for a status code 400, so I guess it's testing that the validation doesn't allow an empty request, but it's also testing that the participant and user queries were called.

Decide what this should be testing, and then focus on testing the outcome you want to see.

Reply:
This test is part of the `hasParticipantAccess` unit test, I want to test the user belongs to participant should go pass middleware. I am using an existing endpoint rather than create a new one. 
I will change 
` expect(res.statusCode).toBe(400);` to ` expect(res.statusCode).not.toBe(403);`